lcpp.lua

```diff
@@ -1377,9 +1375,36 @@ end
 -- @param predefines OPTIONAL a table of predefined variables
 -- @usage out, state = lcpp.compileFile("../odbg/plugin.h", {["MAX_PAH"]=260, ["UNICODE"]=true})
 function lcpp.compileFile(filename, predefines, macro_sources, next, _local)
-	if not filename then error("processFile() arg1 has to be a string") end
-	local file = io.open(filename, 'r')
-	if not file then error("file not found: "..filename) end
+	if type(filename) ~= 'string' then error("processFile() arg1 has to be a string") end

```

Comment:
Following path finding routine seems to be too specific for personal environment (e.g. I have not ever been use C_INCLUDE_PATH or CPLUS_INCLUDE_PATH yet). and I personally think resolving correct file path for *any* environment, is out of scope of this library. Consider patch compileFile to add your proper path finding routine like following. it this not enough for your usage?
```
local orgCompiler = lcpp.compileFile
lcpp.compileFile = function (filename, predefines, macro_sources, nxt, _local)
	filename = your_path_finding_routine(filename)
	return orgCompiler(filename, predefines, macro_sources, nxt)
end
```